packages/core/content-releases/admin/src/components/CMReleasesInjectionZone.tsx

```diff
@@ -0,0 +1,280 @@
+                <Flex direction="column" alignItems="stretch" gap={2}>
+                  <Box paddingBottom={6}>
+                    <SingleSelect
+                      required
+                      label={formatMessage({
+                        id: 'content-releases.content-manager-edit-view.add-to-release.select-label',
+                        defaultMessage: 'Select a release',
+                      })}
+                      placeholder={formatMessage({
+                        id: 'content-releases.content-manager-edit-view.add-to-release.select-placeholder',
+                        defaultMessage: 'Select',
+                      })}
+                      onChange={(value) => setFieldValue('releaseId', value)}
+                      value={values.releaseId}
+                    >
+                      {releases?.map((release) => (
+                        <SingleSelectOption key={release.id} value={release.id}>
+                          {release.name}
+                        </SingleSelectOption>
+                      ))}
+                    </SingleSelect>
+                  </Box>
+                  <FieldLabel>
+                    {formatMessage({
+                      id: 'content-releases.content-manager-edit-view.add-to-release.action-type-label',
+                      defaultMessage: 'What do you want to do with this entry?',
+                    })}
+                  </FieldLabel>
+
+                  <Flex>
+                    <ReleaseActionOptions
+                      selected={values.type}
+                      handleChange={(e) => setFieldValue('type', e.target.value)}
+                    />
+                  </Flex>
+                </Flex>
+              </ModalBody>
+              <ModalFooter
+                startActions={
+                  <Button onClick={handleClose} variant="tertiary" name="cancel">
+                    {formatMessage({
+                      id: 'content-releases.content-manager-edit-view.add-to-release.cancel-button',
+                      defaultMessage: 'Cancel',
+                    })}
+                  </Button>
+                }
+                endActions={
+                  /**
+                   * TODO: Ideally we would use isValid from Formik to disable the button, however currently it always returns true
+                   * for yup.string().required(), even when the value is falsy (including empty string)
+                   */
+                  <Button type="submit" disabled={!values.releaseId} loading={isLoading}>
+                    {formatMessage({
+                      id: 'content-releases.content-manager-edit-view.add-to-release.continue-button',
+                      defaultMessage: 'Continue',
+                    })}
+                  </Button>
+                }
+              />
+            </Form>
+          );
+        }}
+      </Formik>
+    </ModalLayout>
+  );
+};
+
+export const CMReleasesInectionZone = () => {
```

Comment:
You have a typo here, missing "j"

Also I find the name slightly confusing because you're not creating an injection zone, but a component to put inside of that injection zone